setup.py

```diff
@@ -108,6 +108,10 @@ def get_description():
     "pyinstrument",
     "ipython",
 ]
+EXTRAS_REQUIRE["mltools"] = [
```

Comment:
I would suggest splitting this up by ML framework. If someone does `pip install 'coffea[mltools]'` and they install 3GB of packages in addition to the one they want I suspect there will be expressions of unhappiness.